client/my-sites/stats/stats-list/stats-list-actions.jsx

```diff
@@ -77,18 +79,20 @@ const StatsListActions = ( { data, moduleName } ) => {
 		}
 	}
 
-	return actionItems.length ? (
+	return (
```

Comment:
Is rendering the `ul` okay if both `actionItems` and `children` are falsey?

Reply:
Right... hanging empty nodes aren't very useful. I'll address it.